java/code/src/com/redhat/rhn/common/util/DateFormatTransformer.java

```diff
@@ -45,11 +45,11 @@ public String write(Date date) throws Exception {
     }
 
     /**
-     * Parses a datetime from XML format (e.g. 2017-02-22T10:23).
-     * @return a transformer for parsing YYYY-MM-dd'T'HH:mm
+     * Parses a datetime from XML format (e.g. 2017-02-22T10:23:55).
+     * @return a transformer for parsing yyyy-MM-dd'T'HH:mm:ss
```

Comment:
Just a suggestion: maybe we can add a pointer comment to the documentation, as at first glance I was puzzled but then........ https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/text/SimpleDateFormat.html

Reply:
Yeah that was a nasty one to figure out. Agreed I will add the link to the docs.